docs/ManualDeploymentGuideScalarDLOnAzure-1.md

```diff
@@ -0,0 +1,217 @@
+* You must create a new `user node pool` with the name `scalardlpool` for Scalar DL deployment.
+* You must select the subnet other than `k8s_ingress` subnet for creating the AKS cluster.
+* You must create a Kubernetes cluster with `service principal` as the Authentication method.
+* You must create a Kubernetes cluster with `Azure CNI`.
+* You must add a role assignment to the `k8s_ingress` subnet to access the newly created AKS cluster service principal using the role of `Network Contributor`.
+
+### Recommendations
+
+* You should use Kubernetes node size `Standard D2s v3` for Scalar DL node pool.
+* You should create 3 nodes in each node group for high availability in production.
+* You should configure autoscaling for the AKS cluster on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/aks/cluster-autoscaler#set-the-cluster-autoscaler-profile-on-an-existing-aks-cluster).
+
+### Steps
+
+* Create an AKS cluster on the basis of [Azure official guide](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal#create-an-aks-cluster) with the above requirements and recommendations.
+* Configure kubectl to connect to your Kubernetes cluster using the `az aks get-credentials` command.
+
+    ```console
+    $ az aks get-credentials --resource-group myResourceGroup --name myAKSCluster
+    ```
+
+## Step 4. Install Scalar DL
+
+In this section, we will deploy Scalar DL on the AKS cluster using [Helm charts](https://github.com/scalar-labs/helm-charts).
+
+### Prerequisites
+
+You must install Helm on your bastion to deploy helm-charts:
+
+* [Helm](https://helm.sh/docs/intro/install/): Helm command-line tool to manage releases in the AKS cluster, Helm version 3.2.1 or latest is required.
+  In this guide, it is used to deploy Scalar DL and Schema loading helm charts to the AKS cluster.
+
+### Requirements
+
+* You must have the authority to pull `scalar-ledger` and `scalardl-schema-loader` container images.
+* You must confirm that the replica count of the ledger and envoy pods in the `scalardl-custom-values.yaml` file is equal to the number of nodes in the `scalardlpool`.
+
+### Steps
+
+1. Download the following Scalar DL configuration files from [the scalar-kubernetes repository](https://github.com/scalar-labs/scalar-kubernetes/tree/master/conf). Note that they are going to be versioned in the future, so you might want to change the branch to use a proper version.
+    * scalardl-custom-values.yaml
+    * schema-loading-custom-values.yaml
+
+2. Update the database configuration in `scalarLedgerConfiguration` and `schemaLoading` sections as specified in [configure Scalar DL guide](./ConfigureScalarDL.md).
+
+3. Create the docker-registry secret for pulling the Scalar DL images from GitHub Packages.
+    
+   ```console
+    $ kubectl create secret docker-registry reg-docker-secrets --docker-server=ghcr.io --docker-username=<github-username> --docker-password=<github-personal-access-token>
```

Comment:
It would be helpful to describe that the GitHub PAT is required to work with ghcr.io. (In the Prerequisites section?)
And the scopes of the token should also be described. I believe that `read:packages` is only the one needed, right?

Reply:
Thank you for the suggestion, I have added prerequisites in b2c656c.
>  I believe that read:packages is only the one needed, right?

Yes.